wiki/docs/installation-instructions-vm.md

```diff
@@ -1,103 +1,86 @@
 ## How to Install D3A using a Virtual Machine (useful especially on Windows)
 
-
-### Prerequisites
-
-#####  Windows environment settings
-1. Enable [Intel Virtualization](https://stackoverflow.com/a/34305608/13507565) on your computer in [BIOS](https://2nwiki.2n.cz/pages/viewpage.action?pageId=75202968).
-2. Go to your [Windows Features](https://www.windowscentral.com/how-manage-optional-features-windows-10) setting and disable Windows Hypervisor Platform (or Hyper-V) and enable Virtual Machine Platform.
-
-
```

Comment:
@chrisd12 I would suggest to keep it as part of document, as for the some the windows machine its disabled by default and the user manually has to do it.

Reply:
Ah ok I see. I will put it back then! Thanks a lot for reviewing this !